internal/service/alarms/serve.go

```diff
@@ -0,0 +1,130 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"log"
```

Comment:
shouldn't we use the log package of the repo?

Reply:
The project logger file seems to be unused and most of the project for example using `slog` where possible which i agree with. 

I dropped "log" and instead using "slog" everywhere